ingest/src/server_mobile.rs

```diff
@@ -228,7 +239,7 @@ where
             .and_then(|public_key| self.verify_network(public_key))
             .and_then(|public_key| self.verify_signature(public_key, event))
             .map(|(_, event)| CellHeartbeatIngestReportV1 {
-                received_timestamp: timestamp,
+                received_timestamp: timestamp.timestamp_millis() as u64,
```

Comment:
is this change to millisec precision gonna impact anything downstream?

Reply:
no, the first line of the function used to be `let timestamp: u64 = Utc::now().timestamp_millis() as u64;` and I just changed to keep it in `DateTime<Utc>` until written into the proto

Reply:
I see that now